src/p4est_extended.h

```diff
@@ -602,7 +602,9 @@ p4est_t            *p4est_source_ext (sc_io_source_t * src,
  */
 p4est_file_context_t *p4est_file_open_read_ext (sc_MPI_Comm mpicomm,
                                                 const char *filename,
-                                                char *user_string,
+                                                char
+                                                user_string
+                                                [P4EST_NUM_USER_STRING_BYTES],
```

Comment:
Wondering about the [FIXEDSIZE] element. Were we not getting rid of this construction?

Reply:
Yes, but I did not added this to this PR. This PR has only the purpose to fix the darwin ci run and smoothing some other places in the code. The code change above was done to have a consistent user string parameter convention.
Do you want that I include the adjustment of the function interfaces already in this PR?

Reply:
Ah I see. No this can wait, then.